src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
+import { Comment } from '../../types/types';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  list: Comment[];
+  idOfPost: number;
+};
+
+export const CommentList: React.FC<Props> = (props) => {
+  const array = props.list.find(
+    comment => comment.postId === props.idOfPost
+    && <CommentInfo comment={comment} key={comment.id} />,
+  );
+
+  if (array === undefined) {
+    return (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    );
+  }
+
+  return (
+    <>
+      {props.list.map(
```

Comment:
Use the filtered comments when mapping, removing the unnecessary inner condition 🧹.